### DISCUSSION TITLE
ValueError: invalid literal for int() with base 10

### DISCUSSION THREAD

**Discussion by kushnirv** [vb64/django.admin.geomap#41] [Ideas]
If I use id not int() I get an error:
`ValueError: invalid literal for int() with base 10`
In my case I replaced all id in the project with uuid

**Comment by vb64** [comment]
Please change the project settings in the [example directory](https://github.com/vb64/django.admin.geomap/tree/main/example) of the library in the same way as your project setup and create a pull request with these changes.

If this error is raised in tests, then we will try to fix the problem.

**Reply by kushnirv** [reply]
https://github.com/vb64/django.admin.geomap/pull/42

**Reply by vb64** [reply]
Implemented in [version 1.6](https://pypi.org/project/django-admin-geomap/). Thanks for your contribute.

**Comment by vb64** [comment]
Please let me know if the new version works as expected in your case.

**Reply by kushnirv** [reply]
It’s working on my project
